diff --git a/node_modules/mumble-client-codecs-browser/lib/decode-worker.js b/node_modules/mumble-client-codecs-browser/lib/decode-worker.js
index 3925f29..be9af92 100644
--- a/node_modules/mumble-client-codecs-browser/lib/decode-worker.js
+++ b/node_modules/mumble-client-codecs-browser/lib/decode-worker.js
@@ -1,10 +1,6 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
 
-exports.default = function (self) {
   var opusDecoder, celt7Decoder;
   self.addEventListener('message', function (e) {
     var data = e.data;
@@ -55,10 +51,12 @@ exports.default = function (self) {
       }, [_decoded.buffer]);
     }
   });
-};
+
 
 var _libopus = require('libopus.js');
 
 var _libcelt = require('libcelt7.js');
 
 var MUMBLE_SAMPLE_RATE = 48000;
+
+export default null
\ No newline at end of file
diff --git a/node_modules/mumble-client-codecs-browser/lib/decoder-stream.js b/node_modules/mumble-client-codecs-browser/lib/decoder-stream.js
index 6cfda8b..28a9549 100644
--- a/node_modules/mumble-client-codecs-browser/lib/decoder-stream.js
+++ b/node_modules/mumble-client-codecs-browser/lib/decoder-stream.js
@@ -1,9 +1,5 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
-
 var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
 
 var _stream = require('stream');
@@ -12,17 +8,11 @@ var _reusePool = require('reuse-pool');
 
 var _reusePool2 = _interopRequireDefault(_reusePool);
 
-var _webworkify = require('webworkify');
-
-var _webworkify2 = _interopRequireDefault(_webworkify);
-
 var _toArraybuffer = require('to-arraybuffer');
 
 var _toArraybuffer2 = _interopRequireDefault(_toArraybuffer);
 
-var _decodeWorker = require('./decode-worker');
-
-var _decodeWorker2 = _interopRequireDefault(_decodeWorker);
+import DecodeWorker from './decode-worker';
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
@@ -33,7 +23,7 @@ function _possibleConstructorReturn(self, call) { if (!self) { throw new Referen
 function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
 
 var pool = (0, _reusePool2.default)(function () {
-  return (0, _webworkify2.default)(_decodeWorker2.default);
+  return new DecodeWorker();
 });
 // Prepare first worker
 pool.recycle(pool.get());
@@ -48,11 +38,6 @@ var DecoderStream = function (_Transform) {
 
     _this._worker = pool.get();
     _this._worker.onmessage = function (msg) {
-      if (_this._worker.objectURL) {
-        // The object URL can now be revoked as the worker has been loaded
-        window.URL.revokeObjectURL(_this._worker.objectURL);
-        _this._worker.objectURL = null;
-      }
       _this._onMessage(msg.data);
     };
     return _this;
@@ -112,4 +97,5 @@ var DecoderStream = function (_Transform) {
   return DecoderStream;
 }(_stream.Transform);
 
-exports.default = DecoderStream;
\ No newline at end of file
+//exports.default = DecoderStream;
+export default DecoderStream
\ No newline at end of file
diff --git a/node_modules/mumble-client-codecs-browser/lib/encode-worker.js b/node_modules/mumble-client-codecs-browser/lib/encode-worker.js
index f7187ab..c2ebaa3 100644
--- a/node_modules/mumble-client-codecs-browser/lib/encode-worker.js
+++ b/node_modules/mumble-client-codecs-browser/lib/encode-worker.js
@@ -1,10 +1,6 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
 
-exports.default = function (self) {
   var opusEncoder, celt7Encoder;
   var bitrate;
   self.addEventListener('message', function (e) {
@@ -70,7 +66,7 @@ exports.default = function (self) {
       }, [_buffer]);
     }
   });
-};
+
 
 var _libopus = require('libopus.js');
 
@@ -83,3 +79,5 @@ var _toArraybuffer2 = _interopRequireDefault(_toArraybuffer);
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
 var MUMBLE_SAMPLE_RATE = 48000;
+
+export default null
\ No newline at end of file
diff --git a/node_modules/mumble-client-codecs-browser/lib/encoder-stream.js b/node_modules/mumble-client-codecs-browser/lib/encoder-stream.js
index 021f131..eeb9189 100644
--- a/node_modules/mumble-client-codecs-browser/lib/encoder-stream.js
+++ b/node_modules/mumble-client-codecs-browser/lib/encoder-stream.js
@@ -1,9 +1,5 @@
 'use strict';
 
-Object.defineProperty(exports, "__esModule", {
-  value: true
-});
-
 var _createClass = function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ("value" in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; }();
 
 var _stream = require('stream');
@@ -12,13 +8,7 @@ var _reusePool = require('reuse-pool');
 
 var _reusePool2 = _interopRequireDefault(_reusePool);
 
-var _webworkify = require('webworkify');
-
-var _webworkify2 = _interopRequireDefault(_webworkify);
-
-var _encodeWorker = require('./encode-worker');
-
-var _encodeWorker2 = _interopRequireDefault(_encodeWorker);
+import EncodeWorker from './encode-worker'
 
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
@@ -29,7 +19,7 @@ function _possibleConstructorReturn(self, call) { if (!self) { throw new Referen
 function _inherits(subClass, superClass) { if (typeof superClass !== "function" && superClass !== null) { throw new TypeError("Super expression must either be null or a function, not " + typeof superClass); } subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, enumerable: false, writable: true, configurable: true } }); if (superClass) Object.setPrototypeOf ? Object.setPrototypeOf(subClass, superClass) : subClass.__proto__ = superClass; }
 
 var pool = (0, _reusePool2.default)(function () {
-  return (0, _webworkify2.default)(_encodeWorker2.default);
+  return new EncodeWorker();
 });
 // Prepare first worker
 pool.recycle(pool.get());
@@ -46,11 +36,6 @@ var EncoderStream = function (_Transform) {
 
     _this._worker = pool.get();
     _this._worker.onmessage = function (msg) {
-      if (_this._worker.objectURL) {
-        // The object URL can now be revoked as the worker has been loaded
-        window.URL.revokeObjectURL(_this._worker.objectURL);
-        _this._worker.objectURL = null;
-      }
       _this._onMessage(msg.data);
     };
     return _this;
@@ -96,4 +81,5 @@ var EncoderStream = function (_Transform) {
   return EncoderStream;
 }(_stream.Transform);
 
-exports.default = EncoderStream;
\ No newline at end of file
+//exports.default = EncoderStream;
+export default EncoderStream
\ No newline at end of file
